en/02_Development_environment.adoc

```diff
@@ -2,454 +2,416 @@
+image::/images/library_directory.png[]
 
-image::/images/vs_link_input.png[]
+=== tinyobjloader
 
-Enter the names of the Vulkan and GLFW object files:
+Tinyobjloader can be installed with vcpkg like so: vcpkg install tinyobjloader
 
-image::/images/vs_dependencies.png[]
+=== Setting up Visual Studio
 
-And finally change the compiler to support C{pp}17 features:
+==== Setting up a CMake project
 
-image::/images/vs_cpp17.png[]
+Now that you have installed all the dependencies, we can set up a basic
+CMake project for Vulkan and write a little bit of code to make sure that
+everything works.
 
-You can now close the project properties dialog.
-If you did everything right then you should no longer see any more errors being highlighted in the code.
+I will assume that you already have some basic experience with CMake, like
+how variables and rules work. If not, you can get up to speed very quickly with https://cmake.org/cmake/help/book/mastering-cmake/cmake/Help/guide/tutorial/[this tutorial].
 
-Finally, ensure that you are actually compiling in 64 bit mode:
+You can now use the link:/attachments/[attachments] directory in this tutorial
+as a template for your Vulkan projects. Make a copy, rename it to something like `HelloTriangle`
+and remove all the code in `main.cpp`.
 
-image::/images/vs_build_mode.png[]
+Congratulations, you're all set for xref:03_Drawing_a_triangle/00_Setup/00_Base_code.adoc[playing with Vulkan]!
 
-Press `F5` to compile and run the project and you should see a command prompt and a window pop up like this:
+== Linux
 
-image::/images/vs_test_window.png[]
+These instructions will be aimed at Ubuntu, Fedora and Arch Linux users, but
+ you may be able to follow along by changing the package manager-specific
+ commands to the ones that are appropriate for you.
+You should have a compiler that supports C{pp}20 (GCC 7+ or Clang 5+).
+You'll also need You'll also need `cmake`. Most of this can be installed via
+ larger packages such as build-essentials.
 
-The number of extensions should be non-zero.
-Congratulations, you're all set for xref:03_Drawing_a_triangle/00_Setup/00_Base_code.adoc[playing with Vulkan]!
+We recommend using CLion or another IDE; however, as with most things in Linux, GUIs are entirely optional.
 
-== Linux
+=== Vulkan tarball
 
-These instructions will be aimed at Ubuntu, Fedora and Arch Linux users, but you may be able to follow along by changing the package manager-specific commands to the ones that are appropriate for you.
-You should have a compiler that supports C{pp}17 (GCC 7+ or Clang 5+).
-You'll also need `make`.
+The most important parts you'll need for developing Vulkan applications on
+Linux are the Vulkan loader, validation layers, and a couple of command-line
+utilities to test whether your machine is Vulkan-capable:
 
-=== Vulkan Packages
+Download the VulkanSDK tarball from [LunarG](https://vulkan.lunarg.com/).
```

Comment:
This link is wrongly formatted (markdown style), which doesn't work for Asciidoc. Correct format would be url[caption], so https://vulkan.lunarg.com/[LunarG]